pymesync.py

```diff
@@ -83,6 +83,48 @@ def get_times(self, **queries):
             # Request Error
             return e
 
+    def get_projects(self, **kwargs):
+        """
+        get_times([kwargs])
```

Comment:
kwargs isn't an array, it's a dict. To pass an array of things you want ``*args``.